src/fusor/tools.py

```diff
@@ -89,3 +90,14 @@ async def check_data_resources(
     return FusorDataResourceStatus(
         cool_seq_tool=all(cst_status), gene_normalizer=gene_status
     )
+
+
+def get_error_message(e: ValidationError) -> str:
```

Comment:
I think we should handle cases where there are more than one validation errors rather than just grabbing the first

Reply:
oh good idea! I actually didn't know if there could be more than one. I'll get this addressed ASAP!